cmd/mt-split-metrics-by-ttl/main.go

```diff
@@ -12,36 +12,30 @@ import (
 	"github.com/raintank/dur"
 )
 
-var (
-	// flags from metrictank.go, Cassandra
-	cassandraAddrs               = flag.String("cassandra-addrs", "localhost", "cassandra host (may be given multiple times as comma-separated list)")
-	cassandraKeyspace            = flag.String("cassandra-keyspace", "metrictank", "cassandra keyspace to use for storing the metric data table")
-	cassandraConsistency         = flag.String("cassandra-consistency", "one", "write consistency (any|one|two|three|quorum|all|local_quorum|each_quorum|local_one")
-	cassandraHostSelectionPolicy = flag.String("cassandra-host-selection-policy", "tokenaware,hostpool-epsilon-greedy", "")
-	cassandraTimeout             = flag.Int("cassandra-timeout", 1000, "cassandra timeout in milliseconds")
-	cassandraRetries             = flag.Int("cassandra-retries", 0, "how many times to retry a query before failing it")
-	cqlProtocolVersion           = flag.Int("cql-protocol-version", 4, "cql protocol version to use")
-	cassandraCreateKeyspace      = flag.Bool("cassandra-create-keyspace", true, "enable the creation of the metrictank keyspace")
-	cassandraSchemaFile          = flag.String("cassandra-schema-file", "/etc/metrictank/schema-store-cassandra.toml", "File containing the needed schemas in case database needs initializing")
-
-	cassandraSSL              = flag.Bool("cassandra-ssl", false, "enable SSL connection to cassandra")
-	cassandraCaPath           = flag.String("cassandra-ca-path", "/etc/metrictank/ca.pem", "cassandra CA certificate path when using SSL")
-	cassandraHostVerification = flag.Bool("cassandra-host-verification", true, "host (hostname and server cert) verification when using SSL")
+func main() {
+	storeConfig := cassandra.NewStoreConfig()
+	// hard coded to default because those have no effect in the case of this tool anyway
+	storeConfig.WindowFactor = 20
+	storeConfig.OmitReadTimeout = 60
+	storeConfig.ReadConcurrency = 20
+	storeConfig.ReadQueueSize = 100
 
-	cassandraAuth     = flag.Bool("cassandra-auth", false, "enable cassandra authentication")
-	cassandraUsername = flag.String("cassandra-username", "cassandra", "username for authentication")
-	cassandraPassword = flag.String("cassandra-password", "cassandra", "password for authentication")
+	// flags from cassandra/config.go
+	flag.StringVar(&storeConfig.Addrs, "cassandra-addrs", "localhost", "cassandra host (may be given multiple times as comma-separated list)")
```

Comment:
why is this tool stipulating default values and not just relying on the defaults already set?

Reply:
that is what the tool was already doing.  The defaults used by this tool are different to the defaults used by metrictank, so i kept that behaviour.